src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -87,7 +87,8 @@
  *
  *  <p>The tool also accepts multiple .list files. Pass each in with the -vcfs option.</p>
  *
- * @author David Benjamin &lt;davidben@brgoadinstitute.org&gt;
+ *  <p>See documentation for {@link Mutect2} for examples of using a PoN.</p>
```

Comment:
--> See {@link Mutect2} documentation for usage examples.

Reply:
done